libs/ui-toolkit/src/components/rounded-wrapper/rounded-wrapper.tsx

```diff
@@ -0,0 +1,23 @@
+import classnames from 'classnames';
+import type { ReactNode } from 'react';
+
+export interface RoundedWrapperProps {
+  children?: ReactNode;
+  paddingBottom?: boolean;
+}
+
+export const RoundedWrapper = ({
```

Comment:
question (non-blocking): this roundedness is not something i have seen in other components, but perhaps I have missed something. If we aren't going to be doing this outside of tables IMO a prop for key value table would be better. Happy to wait and see how it goes though

Reply:
Hmm yeah ok. I'll keep this as is for now and catch up with Joelle when she's back, and make it a prop as a secondary ticket if necessary.